lib/rules/function-calc-no-unspaced-operator/index.mjs

```diff
@@ -219,46 +147,30 @@ const rule = (primary, _secondaryOptions, context) => {
 							operator,
 						);
 					}
-				} else if (charAfter && charAfter !== ' ') {
```

Comment:
I am surprised that the removal of this block doesn't trigger a regression.
Is this now covered by something else or was it always unnecessary?

Reply:
Yes, this is already covered by:

- https://github.com/stylelint/stylelint/blob/9d5b9f095309e4f957205f7c297f7c83aba02e8f/lib/rules/function-calc-no-unspaced-operator/index.mjs#L113
- https://github.com/stylelint/stylelint/blob/9d5b9f095309e4f957205f7c297f7c83aba02e8f/lib/rules/function-calc-no-unspaced-operator/index.mjs#L119
- https://github.com/stylelint/stylelint/blob/9d5b9f095309e4f957205f7c297f7c83aba02e8f/lib/rules/function-calc-no-unspaced-operator/index.mjs#L138

E.g.

```css
calc(-1px)    /* operatorIndex === 0 */
calc(1px-2)   /* charBefore === 'x' && charAfter === '2' */
calc(1px- 2)  /* charBefore === 'x' */
```

Reply:
https://github.com/stylelint/stylelint/blob/45d9a2d5656caefc8f276082cc0b5a4a43ada230/lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs#L371

seems to be passing indeed.

Reply:
No wait there's an error in the tests :

https://github.com/stylelint/stylelint/blob/45d9a2d5656caefc8f276082cc0b5a4a43ada230/lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs#L373

It should be `expectedOperatorAfterSign` right?

Reply:
https://github.com/stylelint/stylelint/blob/45d9a2d5656caefc8f276082cc0b5a4a43ada230/lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs#L371-L373

seems to be passing indeed.